bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -297,15 +297,15 @@ spec:
   maturity: alpha
   provider:
     name: OpenTelemetry Community
-  version: 0.27.0
+  version: 0.28.0
   webhookdefinitions:
   - admissionReviewVersions:
     - v1
     - v1beta1
     containerPort: 443
     deploymentName: opentelemetry-operator-controller-manager
     failurePolicy: Fail
-    generateName: vopentelemetrycollector.kb.io
+    generateName: vopentelemetrycollectorcreateupdate.kb.io
```

Comment:
@jpkrohling I just realized I need to run `make bundle` after making a change to webhooks (missed these changes in previous PR). Usually, I run `make manifests container container-push deploy` as the manifests are generated and I test the deployed changes before I create the PR.

Reply:
In another PR, could you add a note about it in the contributing.md? Something like: "when the CRD changes, run ...". 

For the Jaeger Operator, we have a target that will fail the build in case `make generate` would yield changes, implying that the PR has changes that impact the auto-generated files. Perhaps we could have something like this here as well?